projects/plugins/boost/app/lib/minify/functions-service.php

```diff
@@ -199,7 +199,7 @@ function jetpack_boost_page_optimize_build_output() {
 	// We can't assume that the root serves the same content as the subdir.
 	$subdir_path_prefix = '';
 	$request_path       = $utils->parse_url( $request_uri, PHP_URL_PATH );
-	$_static_index      = strpos( $request_path, '/_static/' );
+	$_static_index      = strpos( $request_path, '/_jb_static/' );
```

Comment:
Should `jetpack_boost_get_static_prefix()` be used here rather than the string literal?

Reply:
Thank you; good catch. Fixed in fc5405d979.